### PR TITLE
Add http sample and bump deps

### PR DESCRIPTION
**Pull request by yathindrakodithuwakku**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: samples/asgardeo-react-js-app/src/app.js [hunk 1]
```diff
@@ -17,17 +17,21 @@
  */
 
 import React, { useEffect, useState } from "react";
-import * as ReactDOM from "react-dom";
+import { createRoot } from "react-dom/client";
```
Comment (thivi) [inline samples/asgardeo-react-js-app/src/app.js:20]: Are these changes needed?

Comment (yathindrakodithuwakku) [reply samples/asgardeo-react-js-app/src/app.js:20]: Not mandatory. Changed to createRoot to fix the warnings getting from react.
